### PR TITLE
[TEST ONLY] Replacing codegen with boxed for testing

### PR DESCRIPTION
**Pull request by soulitzer**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #64654
* #63878

### CODE REVIEW

File: torch/csrc/jit/frontend/schema_type_parser.cpp [hunk 1]
```diff
@@ -389,6 +389,13 @@ std::pair<TypePtr, c10::optional<AliasInfo>> SchemaTypeParser::parseType() {
       if (container && alias_info) {
         container->addContainedType(std::move(*alias_info));
       }
+      if (alias_info) {
+        if (container) {
+          container->addContainedType(std::move(*alias_info));
+        } else {
+          container = std::move(alias_info);
+        }
+      }
```
Comment (soulitzer) [inline torch/csrc/jit/frontend/schema_type_parser.cpp:398]: workaround for testing only

File: test/cpp/api/autograd.cpp [hunk 2]
```diff
@@ -1119,11 +1119,11 @@ TEST(TestAutogradNotImplementedFallback, ViewOp) {
...
-  // raise on rebase_history when it refreshes grad_fn
-  ASSERT_THROWS_WITH(v1.add_(t), "which does not have a derivative implemented is forbidden");
-  // base should not be aware of the views, so this is still okay
-  b1.add_(t);
-  ASSERT_THROWS_WITH(v1.grad_fn(), "which does not have a derivative implemented is forbidden");
+  // // raise on rebase_history when it refreshes grad_fn
+  // ASSERT_THROWS_WITH(v1.add_(t), "which does not have a derivative implemented is forbidden");
+  // // base should not be aware of the views, so this is still okay
+  // b1.add_(t);
+  // ASSERT_THROWS_WITH(v1.grad_fn(), "which does not have a derivative implemented is forbidden");
```
Comment (soulitzer) [inline test/cpp/api/autograd.cpp:1126]: We expect this to fail

File: torch/testing/_internal/common_methods_invocations.py [hunk 3]
```diff
@@ -6902,17 +6902,17 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
...
-           op=torch.linalg.lstsq,
-           dtypes=floating_and_complex_types(),
-           supports_out=True,
-           sample_inputs_func=sample_inputs_linalg_lstsq,
-           supports_autograd=False,
-           decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
-           skips=(
-               SkipInfo('TestJit', 'test_variant_consistency_jit'),
-           )),
+    # OpInfo('linalg.lstsq',
```
Comment (soulitzer) [inline torch/testing/_internal/common_methods_invocations.py:6905]: Due to https://github.com/pytorch/pytorch/issues/64794

**Comment by albanD** [comment]
This can be closed as we're not going to land it?

**Pull request closed without merging**